app/components/tsx-component.ts

```diff
@@ -42,7 +42,7 @@ export default abstract class TsxComponent<
   }> &
     Readonly<TProps>;
 
-  value: TProps['value'];
+  value: any;
```

Comment:
should this be unknown?

Reply:
Unfortunately, I have an error in some components if I use `unknown` here

Reply:
This should be removed.  `TsxComponent` should not be declaring a `value` attribute.